indigo_app/templates/indigo_api/work_popup.html

```diff
@@ -1,41 +1,24 @@
 <div class="popup-content lh-compact">
   <h6>{{ work.title }}</h6>
-  <p class="text-muted mb-1 wb-all">{{ work.frbr_uri }}</p>
+  <p class="text-muted mb-3 wb-all">{{ work.frbr_uri }}</p>
 
   {% if work.assent_date %}
-    <p class="mb-1">Assented to on {{ work.assent_date|date:"Y-m-d" }}</p>
+    <p class="mb-2">Assented to on {{ work.assent_date|date:"Y-m-d" }}.</p>
   {% endif %}
 
-  <p class="mb-1">Published on {{ work.publication_date|date:"Y-m-d" }}
```

Comment:
Urg, please can we keep these on separate lines! The long line is crazy to review and debug. To handle the whitespace issue before the '.' at the end, move the {% if %} onto the end of the line above:

```
<p class="mb-1">Published on {{ work.publication_date|date:"Y-m-d" }}{%
  if work.publication_name and work.publication_number and work.publication_document %}
    in ... <a>aoeu</a>{%
  endif %}.
</p>
```

Reply:
Thanks :-)
Updated